packages/host/app/commands/get-card.ts

```diff
@@ -0,0 +1,32 @@
+import { service } from '@ember/service';
+
+import type * as BaseCommandModule from 'https://cardstack.com/base/command';
+import { type CardDef } from 'https://cardstack.com/base/card-api';
+
+import HostBaseCommand from '../lib/host-base-command';
+import type StoreService from '../services/store';
+
+export default class GetCardCommand extends HostBaseCommand<
+  typeof BaseCommandModule.GetCardInput,
+  CardDef
+> {
+  @service declare private store: StoreService;
+
+  static actionVerb = 'Get Card';
+  static description = 'Get a card from the store by its ID';
+
+  async getInputType() {
+    let commandModule = await this.loadCommandModule();
+    const { GetCardInput } = commandModule;
+    return GetCardInput;
+  }
+
+  protected async run(input: BaseCommandModule.GetCardInput): Promise<CardDef> {
+    let card = await this.store.get(input.cardId);
```

Comment:
I think @habdelra would want you to include some comments about the expected lifetime of this retrieved card.

Reply:
Absolutely. I’m a very dubious about consuming `getCard()` outside of a component. It’s really meant to be used by a component. We should really talk about the use case here. I think we might be better suited using a different store API.

Reply:
Or my bad I saw this command name and Luke’s and assumed it was wrapping `getCard()`, but I see that it is not. It’s still worth chatting about. But in the way we use the store api here the card returned is eligible for garbage collection and must be immediately consumed. So lifetime is a moot point.

Reply:
Yeah, I'm not sure what the best thing is here, I added this as part of a demo modifying a card because non-host commands have no way of accessing the store. Cards do, but commands don't.

Do we want to wrap the store in a series of commands like this? Or give access to the store to commands as well as cards?

Reply:
I think the concept of exposing this as a command is good. Just needs docs to explain proper use I think.